include/rocksdb/advanced_options.h

```diff
@@ -189,11 +189,10 @@ struct CompressionOptions {
 // Temperature of a file. Used to pass to FileSystem for a different
 // placement and/or coding.
 enum class Temperature : uint8_t {
-  kHot,
-  kWarm,
-  kCold,
-  kTotal,
-  kUnknown = kTotal,
+  kUnknown = 0,
```

Comment:
Should we give a larger range between kHot and kCold for future extension not just 1 to 3?

Reply:
Hmm... I don't know. I can leave some room just in case.